### ISSUE TITLE
Problem with update 

### ISSUE THREAD

**Issue by sergeushenecz** [valor-software/ngx-bootstrap#1544]
I am updating library with 1.1.16-9 to 1.3.0 and I am getting 
index.js:1Uncaught SyntaxError: Unexpected token export
![image](https://cloud.githubusercontent.com/assets/7217517/22237548/ada95eb2-e21d-11e6-8dd5-5d6bd89c2511.png)

Can you help me?

**Comment by valorkin** [comment]
webpack or system.js?
if system.js you should use bundled version of ng2-bootstrap
if webpack it should consume it without issues

http://valor-software.com/ng2-bootstrap/#/
`how to use with` section covers typical configurations with instructions

**Comment by AlexViderman** [comment]
I got the same error once 1.3.0 was released. **When I revert to 1.2.6 the problem goes away without changing anything else**. I am using WebPack 1.14. When I don't uglify I get this run-time error:

![image](https://cloud.githubusercontent.com/assets/21067092/22262858/b8116a18-e22f-11e6-851a-6afc1265f80b.png)

When I uglify, I get this compile-time error:

![image](https://cloud.githubusercontent.com/assets/21067092/22262964/0c8c5ee0-e230-11e6-9f54-5085150a79c6.png)

**Comment by valorkin** [comment]
Oh, thing is, I have to keep my package in sync with @angular, they switched to es6 modules, so do I. Webpack 2 natively supports es6 module format.

**Comment by AlexViderman** [comment]
@valorkin thank you for the quick response. I will work on getting my webpack 1 to support es6 modules.

**Comment by sergeushenecz** [comment]
@valorkin  I use  1.14.0 webpack but 1.1.16-9 version ng2-bootstrap work very nice.
what could be the reason else?

**Comment by valorkin** [comment]
I wrote reason above, all other changes described in change log

**Comment by matejkramny** [comment]
Fixed by replacing `ng2-bootstrap/ng2-bootstrap` to `ng2-bootstrap` as it says in the warnings..

```
console.warn('DEPRECATED: please replace import of `ng2-bootstrap/ng2-bootstrap` with `ng2-bootstrap`');
export * from './index';
```

**Comment by MarkPieszak** [comment]
As @matejkramny mentioned, you need to update your Ng2BootstrapModule import to `from `ng2-bootstrap` get rid of the `/ng2-bootstrap` at the end.

Also don't forget to add: `Ng2BootstrapModule` **`.forRoot()`** <-- the forRoot() when importing the module to your own NgModule.

Still loving the library @valorkin great work as always :100:

Have you featured in my aspnet & angular universal repo [here](https://github.com/MarkPieszak/aspnetcore-angular2-universal) finally was able to upgrade to the 1.3.x version after fixing the workarounds needed to get Universal to play well with  NG 2.4.5.

Works like a charm though, your components are still coming in great server-rendered! :+1: Keep up the great work!

**Comment by AlexViderman** [comment]
I decided to upgrade webpack from 1.4 to 2.2 and the new exports work now. It was time to upgrade to webpack 2 anyway...

**Comment by matejkramny** [comment]
@MarkPieszak can you explain why `.forRoot()` is needed?

I import it in the root app module, and also in other "sub" modules (hope that makes sense!). Do the sub modules also require `.forRoot()` import?

sorry if it doesn't make sense, haven't slept for ~36 hours.

**Comment by valorkin** [comment]
seems to be fixed in 1.3.3

@matejkramny .forRoot() needed to not duplicate providers declarations in lazy loaded modules
